.github/workflows/macos.yml

```diff
@@ -198,6 +198,8 @@ jobs:
           python-version: '3.11.4'
 
       - name: before_script
+        env:
+          MACOS_CERTIFICATES_P12: ${{ secrets.APPLE_CODE_SIGN_IDENTITY }}
```

Comment:
No. The secret is also called `MACOS_CERTIFICATES_P12`.

`APPLE_CODE_SIGN_IDENTITY`, used at signing time, identifies the certificate to use. This item is the actual certificates to be stored in the macOS Keychain.

You also need to set `MACOS_CERTIFICATES_PASSWORD` which is in secrets with the same name to enable `before_build_macos.sh` to import the certificates from MACOS_CERTIFICATES_P12.

Reply:
Certificates are now added correctly to the key chain.